package.json

```diff
@@ -17,6 +17,27 @@
   },
   "homepage": "https://github.com/TechLadies/onesg-app#readme",
   "dependencies": {
-    "express": "^4.17.1"
-  }
+    "express": "^4.17.1",
+    "knex": "^0.21.5",
+    "objection": "^2.2.3",
+    "sequelize": "^6.3.5",
+    "sqlite3": "^5.0.0"
+  },
+  "devDependencies": {
+    "eslint": "^7.10.0",
+    "eslint-config-prettier": "^6.12.0",
+    "eslint-plugin-prettier": "^3.1.4",
+    "eslint-plugin-vue": "^6.2.2",
```

Comment:
Let's remove the vue plugin

Reply:
Also, I think you are missing some dev packages - `lint-staged` & `eslint-config-airbnb`

Reply:
```suggestion
    "eslint-config-airbnb": "^18.2.0",
    "eslint-plugin-import": "^2.22.1",
    "lint-staged": "^10.4.0",
```